prusti-common/src/vir/ast/expr.rs

```diff
@@ -228,6 +243,16 @@ impl fmt::Display for Expr {
                     .collect::<Vec<String>>()
                     .join(", "),
             ),
+            // mimicked DomainFuncApp for Display
+            Expr::BackendFuncApp(ref function, ref args, ref _pos) => write!(
+                f,
+                "{}({})",
```

Comment:
Does the function name always contain e.g. `backend` or something to identify it somewhat uniquely in string dumps?